cluster/vagrant/util.sh

```diff
@@ -288,6 +303,11 @@ function restart-kube-proxy {
   ssh-to-node "$1" "sudo systemctl restart kube-proxy"
 }
 
+# Restart the apiserver
+function restart-apiserver {
```

Comment:
SGTM, just please get around to it. :)
On Feb 9, 2015 5:28 PM, "Rajat Chopra" <notifications@github.com> wrote:

> In cluster/vagrant/util.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/4268#discussion_r24382384>
> :
>
> > @@ -288,6 +303,11 @@ function restart-kube-proxy {
> >    ssh-to-node "$1" "sudo systemctl restart kube-proxy"
> >  }
> >
> > +# Restart the apiserver
> > +function restart-apiserver {
>
> Expediency just. Let this fly and I will port this across so that services
> e2e looks cleaner also.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/4268/files#r24382384>
> .
>